framework/Models/PostProcessors/BasicStatistics.py

```diff
@@ -957,6 +969,40 @@ def __runLocal(self, inputData):
         percentileSet = percentileSet.rename({'quantile':'percent'})
       calculations[metric] = percentileSet
 
+      # because percentile is different, calculate standard error here
```

Comment:
if I recall correctly the document source for the ste of all these metrics is already cited in the code, right? If not, then we should note the source here.

Reply:
The reference was actually missing, I added it to both BasicStatistics and EconomicRatio in commit 063b945.